visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.component.html

```diff
@@ -16,8 +16,8 @@ <h3 class="header-text">Flattened</h3>
 		</md-list-item>
 
 		<md-list-item
-			ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path'] track by $ctrl._viewModel.flatten.indexOf(item)"
-			title="{{::item.path}}"
+			ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path'] "
+			title="{{::item.path}} track by $ctrl._viewModel.flatten.indexOf(item.path)"
```

Comment:
The `track by` should be part of `ng-repeat` :-) similar below

Reply:
putting it in "ng-repeat" brings in the previous problem of uncertain behaviour during the removal of items from the flatten and exclude lists. I suggest we leave it out.. :)

Reply:
Done :)